src/resources/vue/pages/components/SearchBar.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
Dit is een inputfield of een searchbar? Want inputbars willen we denk ik herbruiken?